rust/src/countme.rs

```diff
@@ -102,15 +102,18 @@ pub fn entrypoint(_args: &[&str]) -> Result<()> {
         }
     });
 
-    // Update cookie timestamp only if at least one request is successful
+    println!("Successful requests: {}/{}", successful, repos.len());
     if successful == 0 {
-        bail!("No request successful");
+        // Return false as Result to let the parent know that we should exit
+        // with a specific return value to signal to systemd to retry later.
+        eprintln!("No request successful");
+        return Ok(false);
```

Comment:
You should be able to just change the signature of the countme entrypoint to `Result<i32>` and then here return e.g. `Ok(42)`, and then...

Reply:
Oh, true!